docs/explanations/Cloud-Images-Overview.md

```diff
@@ -0,0 +1,85 @@
+# Ubuntu Cloud Images
+
+Canonical produces new Ubuntu produces new images regularly for various clouds. The Public Cloud team produces, registers, and publishes these images to the Amazon AWS EC2, Amazon AWS EKS, Google GCE, and Microsoft Azure. Other clouds and products, CPC works directly with partners to create a regular release process. These partners include: IBM VPC, IBM Classic, Oracle Cloud Infrastructure (OCI), Google GKE products, and others. CPC also owns the publication of "Premium" images to cloud marketplaces. This includes products such as Ubuntu Pro, Ubuntu Pro Fips, Anbox Cloud, and more. This document will help users understand the different places you can find official Canonical supported Ubuntu images, some of the basic terminology used throughout our docs, and the high level release plan.
```

Comment:
> Canonical produces new Ubuntu produces new

`Canonical produces new Ubuntu images` maybe?

Reply:
> The Public Cloud team produces

`The Public Cloud team (CPC) produces` because later we the term CPC is used.

Reply:
> and publishes these images to the Amazon AWS EC2, Amazon AWS EKS, Google GCE, and Microsoft Azure

We don't publish to Amazon AWS EKS. Maybe you mean AWS Marketplace here?

Reply:
We produce EKS images. I think i'm confused then -- they're EC2 images that can be used as EKS hosts? So i should list it in that fashion?

Reply:
We produce EKS images which are just normal images registered in EC2 (not Marketplace). And when creating a EKS nodegroup, you can select our EKS images (via the AMI ID)